.github/workflows/basic-python3.yml

```diff
@@ -23,6 +23,7 @@ jobs:
           #     gracefully, make them non-daemonic and use a suitable
           #     signalling mechanism such as an Event."
           - pytest --no-cov -k 'not test_BaseType_Unicode and not test_nestedStructure and not testLockedClass'
+          # - pytest --no-cov src/DIRAC/Core/Security/test
```

Comment:
@chaen In the previous Python 3 merge request I had missed this test. I tried adding it here but it fails in a non obvious way. Would you be able to take a look at some point?

(if you can you should work on top of this branch as I fixed all except one of the failures already)

Reply:
Ah you mean the `Security` tests ? Yes, these are nasty, because they unload some modules from memory in order to do a `pyGSI`/`M2Crypto` comparison. I think they can be simplified, and then not run anymore as a specific tests. I'll have a look !

Reply:
So that seems to be an `M2Crypto` issue. I have opened a bug report. I suspect that the problem lies in the C-API, probably some mess with `str`/`bytes` but since I have no knowledge of that, I will wait to see if there is a reaction before embarking in fixing it myself

https://gitlab.com/m2crypto/m2crypto/-/issues/287